sdks/python/apache_beam/metrics/monitoring_infos.py

```diff
@@ -104,7 +109,7 @@ def extract_distribution(monitoring_info_proto):
   return None
 
 
-def create_labels(ptransform='', tag=''):
+def create_labels(ptransform='', tag='', namespace='', name=''):
```

Comment:
Maybe use `None` as default value? Since you're still doing `if property: labels[...] = property`, None is likely a slightly better fit?